src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -85,7 +85,7 @@ private void CallContractInternal(ContractState contract, ContractMethodDescript
         protected internal void CallNativeContract(string name)
         {
             NativeContract contract = NativeContract.GetContract(name);
-            if (contract is null || contract.ActiveBlockIndex > Snapshot.PersistingBlock.Index)
+            if (contract is null || contract.ActiveBlockIndex > Snapshot.Height)
```

Comment:
`Snapshot.Height + 1` ?

Reply:
At that time, the contract is not active.